backend/src/app.ts

```diff
@@ -29,4 +30,11 @@ app.get('/', async (req, res) => {
   res.status(200).send(JSON.stringify(faqs));
 });
 
+app.post('/new-review', async (req, res) => {
+  const doc = reviews.doc();
+  const review: Review = req.body;
```

Comment:
Not sure if we're at this stage yet but we probably should destructure the properties we want. This prevents us from storing extra/non-schema-related properties that attackers might include in the request body.

Reply:
alternatively, you can just do `const review = req.body as Review`